x-pack/plugins/maps/public/connected_components/timeslider/timeslider.tsx

```diff
@@ -32,6 +40,9 @@ interface State {
   range: number;
   timeslice: [number, number];
   ticks: EuiRangeTick[];
+  isPopoverOpen: boolean;
+  step: number;
```

Comment:
`step` state will need to be stored in redux so its value is preserved when the time range changes. That way, the interval can be saved and re-applied if it still makes sense when the time range changes. How about putting `timesliderInterval` into redux at https://github.com/elastic/kibana/blob/main/x-pack/plugins/maps/public/reducers/ui.ts#L32

Reply:
Done